energy-integration/governance-v2/src/proposal.rs

```diff
@@ -87,5 +86,5 @@ pub struct GovernanceProposal<M: ManagedTypeApi> {
     pub proposer: ManagedAddress<M>,
     pub actions: ArrayVec<GovernanceAction<M>, MAX_GOVERNANCE_PROPOSAL_ACTIONS>,
     pub description: ManagedBuffer<M>,
-    pub fees: ProposalFees<M>,
+    pub fee_payment: EsdtTokenPayment<M>,
```

Comment:
As ProposalFees struct is no longer user, I would remove it, along with FeeEntry struct.